content/guides/cli/scripts/deprovision-users.md

```diff
@@ -86,6 +86,13 @@ For example:
      Managed User 1,ManagedUser1@test.com
    ```
 
+Set the user ID which will be the new owner of user content
+   before deleting the user, set the `NewFilesOwnerID` parameter to the
+   user ID you want.
```

Comment:
This sentence is convoluted a bit - should it be:
`To set the user ID which will be the new owner of user content
   before deleting the user, set the NewFilesOwnerID parameter to the
   user ID you want.` 
   
   So basically `To` is missing at the beginning of the sentence?